app/models/katello/content_override.rb

```diff
@@ -0,0 +1,36 @@
+module Katello
+  class ContentOverride
+    attr_accessor :content_label, :name, :value
+    ENABLEMENT_VALUES = ["yes", "no", "0", "1", "default"].freeze
+
+    def initialize(content_label)
+      @content_label = content_label
+    end
+
+    def enabled=(value = nil)
+      @name = "enabled"
+      @value = value
+    end
+
+    def to_hash
+      ret = {"content_label" => @content_label, "name" => @name}
+      ret["value"] = @value if @value
+      ret
+    end
+
+    def self.create_enabled(content_params)
```

Comment:
Why opt for this class method to create an object when we Ruby has the `new` method which can create and return an object that can be used?